cmd/draft/create.go

```diff
@@ -70,14 +70,14 @@ func (c *createCmd) run() error {
 		mfest.Environments[manifest.DefaultEnvironmentName].Name = c.appName
 	}
 
-	chartExists, err := osutil.Exists(filepath.Join(c.dest, "chart"))
+	chartExists, err := osutil.Exists(filepath.Join(c.dest, pack.ChartDir))
```

Comment:
nit: maybe rename `pack.ChartDir` to `pack.ChartsDir` :)

Reply:
We actually have a `pack.ChartsDir` available. I'll quickly refactor this